### PR TITLE
Remove unused comment

### PR DESCRIPTION
**Pull request by akram**
A comment was here, but not relevant

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -126,9 +126,7 @@ PR_COMMIT := $(shell echo $$CLONEREFS_OPTIONS | jq '.refs[0].pulls[0].sha')
 PULL_NUMBER := $(shell echo $$CLONEREFS_OPTIONS | jq '.refs[0].pulls[0].number')
 
 ## -- Static code analysis (lint) targets --
-
 .PHONY: lint
-## Runs linters on Go code files and YAML files - DISABLED TEMPORARILY
```
Comment (otaviof) [inline Makefile]: @akram I think this commend is being used over `help` target. Could you try `make help` and see the results?

Comment (sbose78) [reply Makefile]: I think only "DISABLED TEMPORARILY" should have been removed.

**Comment by otaviof** [comment]
Taking the freedom of closing this PR.

**Pull request closed without merging**